model/src/wav_import_export.F90

```diff
@@ -664,12 +671,45 @@ subroutine export_fields (gcomp, rc)
           ix  = mapsf(isea,1)
           iy  = mapsf(isea,2)
           if (mapsta(iy,ix) == 1) then
-             sw_lamult(jsea) = LAMULT(jsea)
+             sww = atan2(USSHY(jsea),USSHX(jsea)) - UD(isea)
+             alphal = atan( sin(sww) / (                                       &
+                            2.5 * UST(isea)*ASF(isea)*sqrt(dair/dwat)          &
+                          / max(1.e-14, sqrt(USSX(jsea)**2+USSY(jsea)**2))     &
+                          * log(max(1.0, abs(1.25*HSL(ix,iy)/HS(jsea))))       &
```

Comment:
Here, we get a floating point error when HS is zero. It is unclear to me why HS becomes exactly zero at a sea point, but regardless, it looks like this expression should be refactored to ensure numerical stability for cells where denominators are very small.

Reply:
I think we need to set `sw_lamult` to 1 when `HS=0`. So I add this condition in the if statement on Line 673.